toolkit/offchain/tests/integration_tests.rs

```diff
@@ -123,15 +118,29 @@ async fn reserve_management_scenario() {
 	let client = initialize(&container).await;
 	let genesis_utxo = run_init_goveranance(&client).await;
 	let txs = run_init_reserve_management(genesis_utxo, &client).await;
+	let (reference_utxo, v_function_hash) = run_create_v_function_utxo(&client, 100000).await;
 	assert_eq!(txs.len(), 3);
 	let txs = run_init_reserve_management(genesis_utxo, &client).await;
 	assert_eq!(txs.len(), 0);
-	let _ = run_create_reserve_management(genesis_utxo, &client).await;
+	let _ = run_create_reserve_management(genesis_utxo, v_function_hash, &client).await;
 	assert_reserve_deposited(genesis_utxo, INITIAL_DEPOSIT_AMOUNT, &client).await;
 	run_deposit_to_reserve(genesis_utxo, &client).await;
 	assert_reserve_deposited(genesis_utxo, INITIAL_DEPOSIT_AMOUNT + DEPOSIT_AMOUNT, &client).await;
-	let _ = run_update_reserve_settings_management(genesis_utxo, &client).await;
-	assert_mutable_settings_eq(genesis_utxo, UPDATED_MUTABLE_SETTINGS, &client).await;
+	run_release_reserve_funds(genesis_utxo, RELEASE_AMOUNT, reference_utxo, &client).await;
+	assert_reserve_deposited(
+		genesis_utxo,
+		INITIAL_DEPOSIT_AMOUNT + DEPOSIT_AMOUNT - RELEASE_AMOUNT,
+		&client,
+	)
+	.await;
+	assert_illiquid_supply(genesis_utxo, RELEASE_AMOUNT, &client).await;
+	let (new_reference_utxo, new_v_function_hash) =
+		run_create_v_function_utxo(&client, 999000).await;
```

Comment:
There is V-function created in initial transaction. It has t0 = timestamp of the 1st Jan 2025.